src/openrct2/game.h

```diff
@@ -150,7 +150,7 @@ extern float gDayNightCycle;
 extern bool gInUpdateCode;
 extern sint32 gGameCommandNestLevel;
 extern bool gGameCommandIsNetworked;
-extern char gCurrentlyLoadedPath[260];
+extern char gCurrentlyLoadedPath[_MAX_PATH];
```

Comment:
I believe we define MAX_PATH in platform.h. That's the file you'll need to include.